src/pretalx/agenda/management/commands/export_schedule_html.py

```diff
@@ -1,74 +1,230 @@
+    for schedule in event.schedules.all():
+        if schedule.version:  # skip wip version
+            yield schedule.urls.public
+
+
+def urls_event(event):
+    yield event.urls.base
+    yield event.urls.schedule
+    yield from urls_schedule_versions(event)
+    yield event.urls.sneakpeek
+    yield event.urls.talks
+    yield from urls_event_talks(event)
+    yield event.urls.speakers
+    yield from urls_event_speakers(event)
+    yield event.urls.changelog
+    yield event.urls.feed
+    yield event.urls.frab_xml
+    yield event.urls.frab_json
+    yield event.urls.frab_xcal
+    yield event.urls.ical
+
+
+def get_path(url):
+    return urllib.parse.urlparse(url).path
+
+
+def get_content(response):
+    if response.streaming:
+        return b''.join(response.streaming_content)
+    else:
+        return response.content
+
+
+def dump_content(dest, path, content):
+    if path.endswith('/'):
+        path = path + 'index.html'
+
+    path = os.path.join(dest, path.lstrip('/'))
+
+    try:
+        os.makedirs(os.path.dirname(path))
+    except FileExistsError:
+        pass
+
+    with open(path, 'wb') as f:
+        f.write(content)
+
+
+def get_mediastatic_content(url):
+    if url.startswith(settings.STATIC_URL):
+        local_path = settings.STATIC_ROOT / url[len(settings.STATIC_URL):]
+    elif url.startswith(settings.MEDIA_URL):
+        local_path = settings.MEDIA_ROOT / url[len(settings.MEDIA_URL):]
+    else:
+        raise Exception(f"URL is neither static or media file (not within {settings.STATIC_URL} or {settings.MEDIA_URL}): {url}")
+
+    with open(local_path, 'rb') as f:
+        return f.read()
+
+
+def export_event(event, dest):
+    with override_settings(COMPRESS_ENABLED=True, COMPRESS_OFFLINE=True), override_timezone(event.timezone):
+        with fake_admin(event) as get:
+            logging.info("collecting page URLs")
+            urls = [*urls_event(event)]
+            assets = set()
+
+            logging.info(f"exporting {len(urls)} pages")
+            for url in map(get_path, urls):
+                logging.debug(url)
+                content = get(url)
+                dump_content(dest, url, content)
+                assets |= set(map(get_path, find_assets(content)))
+
+            css_assets = set()
+
+            logging.info(f"exporting {len(assets)} assets linked to in HTML")
+            for url in assets:
+                logging.debug(url)
+                content = get_mediastatic_content(url)
+
+                dump_content(dest, url, content)
+
+                if url.endswith('.css'):
+                    css_assets |= set(find_urls(content))
+
+            logging.info(f"exporting {len(css_assets)} assets linked to in CSS")
+            for url in (get_path(urllib.parse.unquote(url)) for url in css_assets):
+                logging.debug(url)
+                content = get_mediastatic_content(url)
+                dump_content(dest, url, content)
+
+
+def rm_rf(path):
```

Comment:
I think a more speaking name would be nicer. `delete_directory` or something like that?